public/app/plugins/panel/geomap/layers/data/geojsonMapper.ts

```diff
@@ -68,30 +77,39 @@ export const geojsonMapper: MapLayerRegistryItem<GeoJSONMapperConfig> = {
       }
     });
 
-    const defaultStyle = new Style({
-      stroke: new Stroke({
-        color: DEFAULT_STYLE_RULE.fillColor,
-        width: DEFAULT_STYLE_RULE.strokeWidth,
-      }),
-    });
+    const styles:StyleCheckerState[] = [];
+    if(config.styles) {
+      for(const r of config.styles) {
+        if(r.style) {
+          const s = await getStyleConfigState(r.style);
+          styles.push({
+            point: s.maker(s.base),
+            poly: polyStyle(s.base),
+            rule: r.rule,
+          });
+        }
+      }
+    }
+    if(true) {
```

Comment:
I think something may be up with our linter / maybe something is messed up locally for you with applying prettier? This didn't seem to break the PR build

Reply:
hymmm -- that is scarry.  but I tihnk you are right!